app/feature/feature-forever/src/main/kotlin/com/hedvig/android/feature/forever/ui/ForeverContent.kt

```diff
@@ -67,6 +71,7 @@ internal fun ForeverContent(
     onRefresh = reload,
     refreshingOffset = PullRefreshDefaults.RefreshingOffset + systemBarInsetTopDp,
   )
+  val locale = uiState.locale ?: Locale.ENGLISH
```

Comment:
We can do `getLocale()` inside a composable https://github.com/HedvigInsurance/android/blob/7e85f0db5781027e32700587c32c517454c95dac/app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/GetLocale.kt#L12 which is always non-null, should be easier to use here.

Reply:
Which makes me think, why do we have a nullable Locale here in the first place? maybe from the old code? `LanguageService.getLocale()` also does not return a nullable locale, so we shouldn't even have this issue.
But yeah, since we can get locale in the composable this way, I think you can probably omit the Locale from inside the UiState completely